### ISSUE TITLE
apio build throws exception for fpga101 

### ISSUE THREAD

**Issue by jelly** [FPGAwars/apio#158]
With the latest release and master from a virtualenv `apio build` throws the following exception

```shell
[jelle@lithium][~/projects/fpga101-workshop/tests/led]%apio build
Board: fpga101
Error: 'NoneType' object has no attribute 'get'
```

This all due to the boards json missing the fpga101 board. Error handling should however be a lot better and indicate what the issue is.

**Comment by jelly** [comment]
Note that the just released beta works, but if any board in apio.ini does not exist in the json file you'll get this obscure error.

**Comment by Jesus89** [comment]
Hi. You are right, this is not a nice error. I noticed that this only happens when the board is configured in the `apio.ini` file and not when it is passed as an argument:

```
$ apio build -b X
Info: ignore apio.ini board
Error: unknown board: X
```

Let's show the same error for this case.

**Comment by Jesus89** [comment]
Closed by aff504f95e8169f65155c87e0562aaabec3e4cd4
